clippy_lints/src/consts.rs

```diff
@@ -41,7 +41,9 @@ pub enum Constant {
     Repeat(Box<Constant>, u64),
     /// A tuple of constants.
     Tuple(Vec<Constant>),
-    /// A literal with syntax error.
+    /// A raw pointer
+    RawPtr(u128),
+    /// A literal with syntax error
```

Comment:
NIT(extreme): We had a cleanup #3767 across rust projects recently. One part of this cleanup was to add a fullstop to comments. That's missing here and on the comment above.

Reply:
Ohhhh! That's a good one to know :)

Reply:
Done and did cleanup on other comments on the same file :D

Reply:
> Done and did cleanup on other comments on the same file :D

Thanks, I didn't look on every comment :D